.github/workflows/windows-build-and-test-ignored.yaml

```diff
@@ -18,8 +18,56 @@ on:
       - NOTICE
       - 'bootstrap*'
 jobs:
+  config:
+    runs-on: ubuntu-latest
+    outputs:
+      build_type: ${{ steps.build_type.outputs.build_type }}
+      pg12_latest: ${{ steps.config.outputs.pg12_latest }}
+      pg13_latest: ${{ steps.config.outputs.pg13_latest }}
+      pg14_latest: ${{ steps.config.outputs.pg14_latest }}
+      pg15_latest: ${{ steps.config.outputs.pg15_latest }}
+
+    steps:
+    - name: Checkout source code
+      uses: actions/checkout@v3
+    - name: Read configuration
+      id: config
+      run: python .github/gh_config_reader.py
+    - name: Set build_type
+      id: build_type
+      run: |
+        if [[ "${{ github.event_name }}" == "pull_request" ]]; then
+          echo "build_type=['Debug']" >>$GITHUB_OUTPUT
+        else
+          echo "build_type=['Debug','Release']" >>$GITHUB_OUTPUT
+        fi
   build:
     runs-on: ubuntu-latest
+    needs: config
+    strategy:
+      fail-fast: false
+      matrix:
+        pg: [ 12, 13, 14, 15 ]
+        os: [ windows-2022 ]
+        build_type: ${{ fromJson(needs.config.outputs.build_type) }}
+        ignores: ["chunk_adaptive metadata"]
```

Comment:
Do we need the parameters that don't influence the check name?

Reply:
Since we don't really build, I'd think these parameters don't influence anything and can be removed.

Reply:
It seems we need them for generating the names. The documentation at [Handling skipped but required checks](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks) is not very clear, but if you look at #5176 that contains only documentation changes you can see that some jobs are stalling because the names do not match. They seem to match the parameters we have above.

It might be possible to remove some parameters: I just copied them from the other file because it is easier to ensure that the same matrix is created if you just cut and paste the code.